Clinical-Genomics/cgweb
# English version of contact page

Contact: @cesve 

How to test:
1. install on stage
1. open OP
1. go to contact page

Expected result:
Contact information exists both in Swedish and in proper english

- [x] Code reviewed by @patrikgrenfeldt 
- [x] Tested by @cesve 
- [x] Deploy OK:d by @patrikgrenfeldt 

This is a minor **version bump** because it expands the interface in a backwards compatible manner
